### DISCUSSION TITLE
Execution Time

### DISCUSSION THREAD

**Discussion by vishalkadam18** [apache/skywalking#12221] [Q&A] [answered]
Dear Team,

Good Afternoon,

Are there any features in the application that display the execution time of methods and classes at the UI level ??

Regards,
Vishal Kadam.

**Comment by wu-sheng** [comment, marked as answer]
It is impossible to trace all that. The cost of tracing is more than the original method itself.

You could
- Learn what is profiling, some agents provide that.
- Add custom tracing(such as Java agents) for specific methods.
